### ISSUE TITLE
Price checking

### ISSUE THREAD

**Issue by MrHillyCraft** [andrewda/node-steam-guide#37]
It would be very helpful if you show how to do price checking from database!
Thanks,

**Comment by andrewda** [comment]
Thanks for the suggestion! I could definitely add some price lookup sections using websites like http://csgo.steamlytics.xyz/ – is that about what you're looking for?

**Comment by xmwx38** [comment]
- [Steamlytics SDKs](http://steamlytics.xyz/sdk)
  - [PHP](https://github.com/waylaidwanderer/Steamlytics-SDK-PHP) (official)
  - [Javascript](https://github.com/Rock48/node-steamlytics) (unofficial)
- [Steamlytics APIs](http://steamlytics.xyz/api)
  - [Steam API](http://steam.steamlytics.xyz/api) for `inventory` and `currencies`.
  - [CS:GO API](http://csgo.steamlytics.xyz/api) for `items`, `prices`, and `pricelist`.

**Comment by Alevale** [comment]
Yep, adding that is one of the best ideas, so that you don't really have to make many calculations on what the prices are and so on.. :)

**Comment by xmwx38** [comment]
- [CSGOFAST API](https://api.csgofast.com/price/all)
- [BitSkins API](https://bitskins.com/api)
  - [get_all_item_prices](https://bitskins.com/api/#get_all_item_prices)

**Comment by xmwx38** [comment]
- [Csgobackpack API](http://csgobackpack.net/api/)
- [OPSkins API v2](https://opskins.com/kb/api-v2)
  - [SDK - Node.js](https://www.npmjs.com/package/@opskins/api) (official)

**Comment by DentFuse** [comment]
This can also be used to get price info directly from steam without any restrictions : 
`http://steamcommunity.com/market/priceoverview/?appid=730&market_hash_name={market_hash_name}`
Result for M4A1-S | Decimator (Field-Tested) would be : 
`{"success":true,"lowest_price":"$5.44","volume":"468","median_price":"$5.54"}`

**Comment by andrewda** [comment]
Price checking will definitely be included in the new guide. See #100.
